common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -25,6 +25,7 @@
     get_or_create_sjson,
     generate_sjson_for_all_speeds,
     get_video_transcript_content,
+    get_transcript_from_contentstore,
```

Comment:
shouldn't we need to use `get_transcript` instead of `get_transcript_from_contentstore`?

Reply:
Nope, I am only aiming for contentstore part of the interface.